test/functional/fetchers_test.rb

```diff
@@ -1,16 +1,16 @@
 require "functional/helper"
 
 describe "the fetchers" do
-  parallelize_me!
   include FunctionalHelper
 
   let(:looks_like_a_stacktrace) { %r{lib/inspec/.+\.rb:\d+:in} }
   let(:invocation) { "exec #{path} --no-create-lockfile" }
   let(:run_result) { inspec(invocation) }
+  let(:fetcher_profiles) { "#{profile_path}/fetcher-failures" }
 
   # Refs #4726
   describe "when fetchers fetch a bad dependency" do
-    let(:fetcher_profiles) { "#{profile_path}/fetcher-failures" }
+    parallelize_me!
```

Comment:
We can't parallelize this whole suite?

Reply:
I believe we can, yes. That was a holdover from early work.